ColorPatterns/Scenes/Game/GameController.swift

```diff
@@ -52,6 +52,9 @@ final class GameController: UIViewController {
         viewModel.didSwipeDown()
         updateColorBars()
     }
+    @IBAction func didSwipeUp(_ sender: UISwipeGestureRecognizer) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/d8b255110f7fe5e0ab888f9e24f2205861e86e4e/ColorPatterns/Scenes/Game/GameController.swift#L55"></span>IBActions should be private.<br />
<code>private_action</code> <code>GameController.swift:55</code></td>
    </tr>
  </tbody>
</table>